pages/about.html

```diff
@@ -49,7 +49,6 @@
       </nav>
     </header>
     <section class="headline-section headline-about">
-      <span id="hello-text">"Hello, Sharing World!</span>
       <h1 class="h1-text">
         JAVASCRIPT CAMP KPLA UGANDA <br />
         2023
```

Comment:
- [ ] Good job styling your footer :+1:, but as you can see below it is required from the template to have a dark background on the About page.

**Your design**

![Screenshot from 2023-02-09 05-08-53](https://user-images.githubusercontent.com/105117832/217783064-99d528fb-6a03-4841-b910-1461476b7cc1.png)

**Template**

![Screenshot from 2023-02-07 08-10-00](https://user-images.githubusercontent.com/105117832/217783189-6d327f29-8470-4002-88ed-08c624e60187.png)

_I place this comment here because there are no changes in the specific section, but you have to look inside this document_